nuxt.config.js

```diff
@@ -67,6 +67,10 @@ export default {
     //'~/plugins/vue-moment.client.js',
     '~/plugins/vue-sticky-directive.client.js',
     '~/plugins/vuejs-paginate.client.js',
+    {
```

Comment:
@rsalunga29 FYI https://nuxtjs.org/guide/plugins/#name-conventional-plugin. I have rename the file.

Reply:
Thank you! I'll take a note of this next time :)